package.json

```diff
@@ -32,8 +32,8 @@
     "pretest": "rimraf coverage && npm run lint",
     "test": "npm run test:cover && npm run test:integration && npm run test:browser-local && npm run test:size",
     "test:ci": "./node_modules/contentful-sdk-core/bin/test-ci.sh",
-    "test:cover": "BABEL_ENV=test babel-node  --extensions \".ts\" --extensions \".js\" ./node_modules/istanbul/lib/cli.js cover ./test/runner.js | tap-dot",
-    "test:unit": "BABEL_ENV=test babel-node --extensions \".ts\" --extensions \".js\" ./test/runner.js | tap-dot",
+    "test:cover": "BABEL_ENV=test babel-node  --extensions \".ts\" --extensions \".js\" ./node_modules/istanbul/lib/cli.js cover ./test/runner.js | tap-spec",
```

Comment:
What that change back to `tap-spec`?

Reply:
I simply couldn't correlate the error and the test which failed. IMO, `spec` works better. Could you elaborate on why `dot` is better?

Reply:
If it's not reporting errors than the reporter is just plain bad :) Usually I see dot reporters superior because it reduces the "noise" of all the working tests and simply highlights the ones failing so you CAN actually see them easily. But again, if tap-dot is not living up to that promise for you, just use tap-spec.

Reply:
so, do you think we should revert changes here and leave `tab-spec` installed so we can debug using it?

Reply:
Just move forward for now and maybe you can show me 1on1 what the actual problem is with tab-spec.